packages/lib/server-only/public-api/create-api-token.ts

```diff
@@ -0,0 +1,35 @@
+import { prisma } from '@documenso/prisma';
+
+// temporary choice for testing only
+import { ONE_YEAR } from '../../constants/time';
+import { alphaid } from '../../universal/id';
+import { hashString } from '../auth/hash';
+
+type CreateApiTokenInput = {
+  userId: number;
+  tokenName: string;
+};
+
+export const createApiToken = async ({ userId, tokenName }: CreateApiTokenInput) => {
+  const apiToken = `api_${alphaid(16)}`;
```

Comment:
Do you think we can replace the `api_` with `dk_`, representative of Documenso Key. Feel free to ignore this, just a thought that occured to me.

Reply:
Thanks for the suggestion! I'm leaning towards the `api_` suffix, though, since I'm used to this format. But I'm happy with whatever we decide as a team. 🙌

Reply:
I'd stick with `api_` as well since dk could also be domain key or something relating to dkim